rust/decentraland-godot-lib/src/content/profile.rs

```diff
@@ -0,0 +1,58 @@
+use std::sync::Arc;
+
+use ethers::types::H160;
+
+use crate::{
+    comms::profile::UserProfile,
+    godot_classes::dcl_global::DclGlobal,
+    http_request::{
+        http_queue_requester::HttpQueueRequester,
+        request_response::{RequestOption, ResponseType},
+    },
+};
+
+pub(crate) fn prepare_request_requirements() -> (String, String, Arc<HttpQueueRequester>) {
+    let global = DclGlobal::singleton();
+    let realm = global.bind().get_realm();
+    let lamda_server_base_url = realm.bind().get_lambda_server_base_url().to_string();
+    let profile_base_url = realm.bind().get_profile_content_url().to_string();
+    let http_requester = global
+        .bind()
+        .get_http_requester()
+        .bind()
+        .get_http_queue_requester();
+
+    let lamda_server_base_url = if lamda_server_base_url.is_empty() {
+        "https://peer.decentraland.org/lambdas/".to_string()
```

Comment:
can we add consts like:

https://github.com/decentraland/godot-explorer/blob/9cc068ec6fa9a77691a91cc3db662fce18e5161b/rust/decentraland-godot-lib/src/godot_classes/dcl_realm.rs#L39-L45

to make it easier in the future to change it?

note about the func that I referred: self is needed on the future to access the parameter problem